services/web-app/pages/contributing/schema.mdx

```diff
@@ -192,17 +212,27 @@ asset key to override the inherited default.
 Assets are reusable units of work that are used in products and digital experiences. Every asset
 belongs to a library.
```

Comment:
Suggest giving examples of assets for a softer intro: Assets include re-usable resources such as components, patterns, functions, and templates. Each asset has code, designs, and usage guidance.

(feel free to cut that last sentences as I know it's not always true).